src/rmats2sashimiplot/rmats2sashimiplot.py

```diff
@@ -63,37 +72,39 @@ def checkout(parser, options):
     """
     # bam files and sam files are alternative, the same for the case of events_file and coordinate
     # events_file should be provided together with event_type
-    if (options.s1 is None and options.b1 is None) or (options.s2 is None and options.b2 is None):
-        parser.error("Not enough arguments! Please provide sam or bam files.")
+    if options.s1 is None and options.b1 is None:
```

Comment:
`if (options.s1 is None and options.b1 is None) and (options.s2 is None and options.b2 is None):`